content/secrets/04-walkthrough-step-01/copilot.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Embedded tab content"
+disableToc: true
+hidden: true
+---
+
+In this section, it is important to match the names as described in order for the tutorial to work (so that it matches the format in the repository)
+
+Application Name: `ecsworkshop`
+
+Workload Type: `Load Balanced Web Service`
+
+Service Name: `todo-app`
+
+Deploy local test environment: `yes`
+
+
```

Comment:
mention to use the default './Dockerfile' because that is another prompt from copilot.